frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -151,28 +195,41 @@ export default ({
         [PROPOSAL_GENERIC]: () => L('TODO: Change [generic] from [current] to [new-value]', {})
       }[this.proposalType]()
     },
+    humanDate () {
+      const date = new Date(this.proposal.meta.createdDate)
+      const offset = date.getTimezoneOffset()
+      const minutes = date.getMinutes()
+      date.setMinutes(minutes + offset)
+      const locale = navigator.languages !== undefined ? navigator.languages[0] : navigator.language
+
+      return date.toLocaleDateString(locale, {
+        year: 'numeric', month: 'long', day: 'numeric'
+      })
+    },
     statusDescription () {
       const votes = Object.values(this.proposal.votes)
       const yay = votes.filter(v => v === VOTE_FOR).length
       const nay = votes.filter(v => v === VOTE_AGAINST).length
+      const date = this.humanDate
       const total = yay + nay
       switch (this.proposal.status) {
         case STATUS_OPEN: {
           const excludeVotes = this.proposalType === PROPOSAL_REMOVE_MEMBER ? 1 : 0
 
-          return L('{count} out of {total} members voted.', {
+          return L('{count} out of {total} members voted on {date}', {
             count: votes.length,
-            total: this.groupMembersCount - excludeVotes
+            total: this.groupMembersCount - excludeVotes,
+            date
           })
         }
         case STATUS_FAILED: {
-          return L('Proposal refused with {nay} against out of {total} total votes.', { nay, total })
+          return L('Proposal refused with {nay} against out of {total} total votes on {date}', { nay, total, date })
         }
         case STATUS_CANCELLED: {
           return L('Proposal cancelled.')
         }
         case STATUS_PASSED: {
-          return L('Proposal accepted with {yay} in favor out of {total} total votes.', { yay, total })
+          return L('Proposal accepted with {yay} in favor out of {total} total votes on {date}', { yay, total, date })
```

Comment:
The human strings that we need are as follows (and yes, I realize that not every Figma design is updated with these, I will copy a link to this comment to Figma as well for consistency, cc @leihla):

**For newly opened proposals:**

- `{count} out of {total} members voted. Expires on {date}.`

Note that this `date` should be **the date the proposal expires.**

**For proposals that have passed:**

- `Proposal accepted on {date} with {yay} in favor out of {total} total votes.`

This `date` should be **the date the proposal passed.**

You will need to modify the `group.js` contract process function for `'gi.contracts/group/proposalVote'`. Add the line `Vue.set(proposal, 'dateClosed', meta.createdDate)` here:

```js
        // see if this is a deciding vote
        const result = votingRules[proposal.data.votingRule](state, proposal.data.proposalType, proposal.votes)
        if (result === VOTE_FOR || result === VOTE_AGAINST) {
          // handles proposal pass or fail, will update proposal.status accordingly
          proposals[proposal.data.proposalType][result](state, message)
          Vue.set(proposal, 'dateClosed', meta.createdDate)
        }
```

Then retrieve that value here and display it with the `humanDate` function from `time.js` with `year: 'numeric', month: 'long', day: 'numeric'`.

**For proposals that have passed:**

- `Proposal rejected on {date} with {nay} against out of {total} total votes.`

The `{date}` here should likewise be the same `dateClosed` as above.